typescript-sdk/src/client-sdk/tracing/__tests__/create-tracing-proxy.unit.test.ts

```
@@ -0,0 +1,715 @@
+      expect(span?.ended).toBe(true);
+    });
+
+    it("should handle async errors with finally blocks", async () => {
+      class FinallyClass {
+        public async withFinally() {
+          try {
+            await createDelayedPromise("work", 5);
+            throw new Error("Error in try block");
+          } finally {
+            // This should still execute
+            return "finally result";
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)